WalletWasabi/Helpers/Startup/MacOsStartupChecker.cs

```diff
@@ -0,0 +1,37 @@
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+using WalletWasabi.Microservices;
+
+namespace WalletWasabi.Helpers
+{
+	public static class MacOsStartupChecker
+	{
+		private const string ListCmd = "osascript -e \' tell application \"System Events\" to get every login item\'";
+
+		internal static async Task<bool> CheckLoginItemExistsAsync()
```

Comment:
Why `internal`?

Reply:
VS generated the method and I forgot to change it. Good catch!